server/app.js

```diff
@@ -3,10 +3,19 @@ const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+const mongoose = require("mongoose");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 
+// Connect to the database
+const mongoDB = "mongodb://127.0.0.1/mail-sender-dev";
+mongoose.connect(mongoDB, { useNewUrlParser: true, useUnifiedTopology: true }).then(() => {
+  console.log("Connected to database...");
+});
+const db = mongoose.connection;
+db.on("error", console.error.bind(console, "MongoDB connection error:"));
+
 const { json, urlencoded } = express;
 
 var app = express();
```

Comment:
Now that we are using const, this should really be const (I know you didnt write this loll)